packages/decompress/decompress.0.3/descr

```diff
@@ -0,0 +1,3 @@
+Pure OCaml implementation of Zlib
+
+Pure OCaml implementation of Zlib
```

Comment:
Might be better to not repeat that sentence. Merging anyway, you are welcome to create an other PR if you want to fix it